lib/orm/models.js

```diff
@@ -78,19 +124,11 @@ function shouldAddModel(mirageSchema, type) {
  * @param {{graphQLSchema: Object, mirageSchema: Object}} options
  */
 export function ensureModels({ graphQLSchema, mirageSchema }) {
-  const graphQLSchemaQueryTypes = [
-    graphQLSchema.getMutationType(),
-    graphQLSchema.getQueryType(),
-    graphQLSchema.getSubscriptionType(),
-  ];
-  const typeMap = graphQLSchema.getTypeMap();
-
-  Object.keys(typeMap).forEach(function (typeName) {
-    const { type } = unwrapType(typeMap[typeName]);
-    const isQueryType = graphQLSchemaQueryTypes.includes(type);
+  const models = createModels({ graphQLSchema });
 
-    if (shouldAddModel(mirageSchema, type) && !isQueryType) {
-      ensureModel(mirageSchema, type);
+  models.forEach((model) => {
```

Comment:
So sorry, one more spot where we could add the tiniest consistency by replacing the arrow function with the more traditional one.

Reply:
Ahh, no problem at all. I totally missed this one. Should be good now.